vmm/task/src/config.rs

```diff
@@ -59,6 +62,10 @@ impl TaskConfig {
             .map_err(io_error!(e, "failed to open /proc/cmdline"))?;
         let params: Vec<&str> = cmdline.split_ascii_whitespace().collect();
         for p in params {
+            if p == TASK_DEBUG {
+                config.debug = true;
+                continue;
+            }
```

Comment:
Shall we keep the parse the `TASK_DEBUG` param like the following `parase_cmdline` macro?

Reply:
fixed